packages/flutter/test/services/fake_platform_views.dart

```diff
@@ -272,25 +273,27 @@ class FakeAndroidMotionEvent {
 }
 
 class FakeUiKitView {
-  FakeUiKitView(this.id, this.type);
+  FakeUiKitView(this.id, this.type, [this.creationParams]);
 
   final int id;
   final String type;
+  final Uint8List creationParams;
 
   @override
   bool operator ==(dynamic other) {
     if (other.runtimeType != FakeUiKitView)
       return false;
     final FakeUiKitView typedOther = other;
     return id == typedOther.id &&
-      type == typedOther.type;
+      type == typedOther.type &&
+      creationParams == typedOther.creationParams;
```

Comment:
trivial nit: this would look slightly better if the leading identifiers on each line were lined up

Reply:
done